cilium-ubuntu-next.json

```diff
@@ -105,13 +105,13 @@
       "execute_command": "echo 'vagrant' | {{ .Vars }} sudo -E -S bash '{{ .Path }}'",
       "expect_disconnect": true,
       "scripts": [
-          "provision/ubuntu/vboxsf-next.sh",
           "provision/vagrant.sh",
           "provision/ubuntu/install.sh",
           "provision/golang.sh",
           "provision/swap.sh",
           "provision/registry.sh",
-          "provision/pull-images.sh"
+          "provision/pull-images.sh",
```

Comment:
I would not add the pull images in net-next, It adds more than 7GB in the image, and If we can I prefer to not use that space in a net-next box.